next.config.js

```diff
@@ -3,6 +3,15 @@ const nextConfig = {
   experimental: {
     appDir: true,
   },
+  async redirects() {
+    return [
+      {
+        source: '/tickets',
+        destination: 'https://qkt.io/w31vDE',
```

Comment:
How possible is it read this value from environment variables?

Reply:
Would be nice to make it easy to delete code on tickets...if it's from a central place

Reply:
I agree with you @HassanBahati. I think that will be it

Reply:
what do you mean by deleting code on tickets?

Reply:
Deleting code...I mean...if incase there is a change....there shouldnt so many places to change...placing the actual link in so many places means all those will need to change.....best to have the actual link in one place and reference it by variables in all these places

Reply:
That's why was suggesting to have it as a variable